### ISSUE TITLE
Support of MaterialUI 1.0

### ISSUE THREAD

**Issue by ludoo0d0a** [maccuaa/formsy-mui#1]
Is the support of [Material UI 1.0](https://github.com/callemall/material-ui/blob/155d98497185f291250eebb829b897dc2540c66a/CHANGELOG.md) will be supported ?

**Comment by maccuaa** [comment]
Yes.  I've been working on getting some proper documetation up.  Once that's done I'm going to work on trying out Material 1.0 and and [formsy-react@beta](https://github.com/christianalfoni/formsy-react/tree/beta)
